android/src/main/java/com/lwansbrough/RCTCamera/RCTCameraModule.java

```diff
@@ -54,11 +59,27 @@
 
     private final ReactApplicationContext _reactContext;
 
+    private MediaRecorder mediaRecorder = new MediaRecorder(); 
```

Comment:
Non-public, non-static field names start with m.

http://stackoverflow.com/questions/2092098/why-do-most-variables-in-android-tutorial-start-with-m

Not that it matters much ;)